DESCRIPTION

```diff
@@ -14,7 +14,8 @@ Imports:
     dplyr,
     tidyverse,
     data.table,
-    RSQLite
+    RSQLite,
+    sommer
```

Comment:
Remove packages if you aren't using them.

Reply:
done